dev-python/ciso8601/metadata.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>root@gentoo.org</email>
```

Comment:
Is this really your email address? :)

You should mark yourself as proxied and add the proxy-maint project too:

```xml
  <maintainer type="person" proxied="yes">
    <email>you@your-real-email.com</email>
    <name>Ruslan Kh.name>
  </maintainer>
  <maintainer type="project" proxied="proxy">
    <email>proxy-maint@gentoo.org</email>
    <name>Proxy Maintainers</name>
  </maintainer>
```